--- origsrc/epdfview-0.1.7/src/Makefile.am	2009-02-28 11:38:53.000000000 -0600
+++ src/epdfview-0.1.7/src/Makefile.am	2009-08-04 22:45:12.381861900 -0500
@@ -58,7 +58,7 @@ epdfview_SOURCES =	\
 	main.cxx
 
 libshell_cxxflags = -I$(top_srcdir)/src/gtk $(GTK2_CFLAGS)
-libshell_ldadd = $(GTK2_LIBS) $(top_builddir)/src/gtk/libshell-gtk.a
+libshell_ldadd = $(top_builddir)/src/gtk/libshell-gtk.a $(GTK2_LIBS)
 
 epdfview_CXXFLAGS =	\
 	-DLOCALEDIR='"$(datadir)/locale"'	\
